src/components/molecules/TermList/term-list.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import TermListItem from '../TermListItem';
+
+const TermList = ({
+	searchTerm,
+	termLinkPath,
+	termLinkTrackingHandler,
+	terms,
+	totalTermCount,
+}) => {
+	return (
+		<>
+			<h4> {`${totalTermCount} results found for: ${searchTerm}`} </h4>
+			<dl>
+				{terms.map((term, index) => {
+					return (
+						<TermListItem
+							itemIndex={index+1}
+							key={index}
+							prettyUrlName={term.prettyUrlName}
+							term={term.name}
+							termDefinition={term.definition.html}
+							termId={term.termId}
+							termLinkTrackingHandler={termLinkTrackingHandler}
+							termLinkPath={termLinkPath}
+						/>
+					);
+				})}
+			</dl>
+		</>
+	);
+};
+
+TermList.propTypes = {
+	searchTerm: PropTypes.string.isRequired,
+	termLinkPath: PropTypes.any.isRequired,
+	termLinkTrackingHandler: PropTypes.func,
+	terms: PropTypes.array.isRequired,
```

Comment:
might be helpful to expose the shape of the terms object here or at least rough out what is required in the object

Reply:
👍

Reply:
Done!